config/jobs/cert-manager/cert-manager-periodics.yaml

```diff
@@ -159,6 +162,9 @@ periodics:
       env:
       - name: K8S_VERSION
         value: "1.17"
+      # Disable CertificateSigningRequest e2e tests for pre v1.18 clusters
+      - name: FEATURE_GATES
+        value: ""
```

Comment:
Should this not be `ExperimentalCertificateSigningRequestControllers=false`?
The empty string would be overriden in https://github.com/jetstack/cert-manager/blob/master/devel/addon/certmanager/install.sh#L26 I guess

Reply:
Yep, thanks!